relay.d.ts

```diff
@@ -22,22 +22,22 @@ export declare const Base64ToUInt8UserID: (userId: string) => Uint8Array;
 
 export declare class Relay {
   public ws: WebSocket;
-  public onClientDisconnect: ((userId: Uint8Array) => void) | null;
+  public onClientDisconnect: ((userId: Uint8Array) => unknown) | null;
```

Comment:
This is technically wrong in the sense that relay.js is the only code that should ever be calling the callbacks and it isn't expecting any return.  It wouldn't break the use of the library, but I think `void` is more accurate than `unknown`.

Reply:
This is true for the other return type changes also.